projects/Sensory_interface/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+    <title> Sensory interface </title>
+    <head>
+        <script src="app.js"></script>
+    </head>
+    <body>
+        <textarea id = "textInput"></textarea>
```

Comment:
A default value has been added to the textarea tag, it's looks ugly a bit in the HTML code. A commit will come shortly.

Reply:
Thanks.
If you want to make it prettier, I think you can revert this change, and instead put it in the top of `app.js`.
I'm not sure about the loading order, because you need `textInput` to exist, and you're loading your script inside `head`, so to make that work you might need to move the script to `body` or something like that (not sure).

Anyway, I don't mind, I'm just telling you this because you wrote that you think it looks ugly :)

Reply:
Let's leave it as it is for now, if we keep the textarea in future versions, then we will consider to modify it. 
Thanks!